manifests/m/MediaHuman/LyricsFinder/1.5.6/MediaHuman.LyricsFinder.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MediaHuman.LyricsFinder
+PackageVersion: 1.5.6
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Maybe try using x64 OS to see if it fixes `Validation-Executable-Error` otherwise an engineer will need to take a look.